gc_ruboconfig.gemspec

```diff
@@ -2,14 +2,14 @@
 
 Gem::Specification.new do |spec|
   spec.name          = 'gc_ruboconfig'
-  spec.version       = '2.33.0'
+  spec.version       = '2.34.0'
```

Comment:
this is breaking change (if you update the version you won't get any of the rails setup and you need to manually configure it)

Reply:
Done!